lib/txgh/app.rb

```diff
@@ -20,6 +21,25 @@ def initialize(app = nil)
       200
     end
 
+    get '/config' do
+      config = Txgh::KeyManager.config_from_project(params[:project_slug])
+      branch = Utils.absolute_branch(params[:branch])
+
+      begin
+        tx_config = Txgh::KeyManager.tx_config(
+          config.transifex_project, config.github_repo, branch
+        )
+
+        data = tx_config.to_h
+        data.merge!(branch_slug: Utils.slugify(branch)) if branch
+
+        status 200
+        json data: data
+      rescue => e
+        status 404
```

Comment:
Good catch. This was here originally to indicate when the tx.config file isn't found in git.